utils/runtime-fuzzer/src/lib.rs

```diff
@@ -16,8 +16,12 @@
 // You should have received a copy of the GNU General Public License
 // along with this program. If not, see <https://www.gnu.org/licenses/>.
 
+#![allow(clippy::items_after_test_module)]
```

Comment:
?

Reply:
clippy will warn because of items (functions, in this case) being defined after `mod tests`
https://rust-lang.github.io/rust-clippy/master/index.html#/items_after_test_module